docs/src/main/asciidoc/gradle-tooling.adoc

```diff
@@ -359,6 +359,16 @@ When building an Uber-Jar you can specify entries that you want to exclude from
 
 The entries are relative to the root of the generated Uber-Jar. You can specify multiple entries by adding extra `--ignored-entry` arguments.
 
+[[multi-module-gradle]]
+=== Working with multi-module projects
+
+If your project is a multi-module project, each module that didn't use the Quarkus Gradle plugin needs a way to generate a Jandex index to enable CDI bean discovery.
```

Comment:
Or use `META-INF/beans.xml`

Reply:
See above

Reply:
Hum, this one is more sensitive as the Gradle plugins are unofficial I would prefere to advise using the `beans.xml` trick.